routes/index.js

```diff
@@ -15,9 +15,26 @@ router.get('/', function(req, res) {
 
 /* GET /notebooks - list of notebooks */
 router.get('/notebooks', function(req, res) {
-    res.render('index', {
-        title: 'Notebooks'
-    });
+    nbstore.getNotebooks().then(
+        function success(notebooks) {
+            //if notebook index exists redirect to it immediately
+            var notebookIndex = "index.ipynb"
+            if(notebooks.indexOf(notebookIndex) > -1) {
+                res.redirect('/notebooks/' + notebookIndex);
+            }
+            else {
+                //render a list of all notebooks
+                res.render('index', {
+                    username: req.session.username,
+                    notebooks: notebooks
+                });
+            }
+        },
```

Comment:
Spoke with @jhpedemonte and in api.js we have a [check that the notebook path header matches](https://github.com/jupyter-incubator/dashboards_nodejs_app/blob/master/routes/api.js#L137) ```var matches = notebookPathHeader.match(/^\/notebooks\/(.*)$/);```  resulting in an error when we simply render on the index ```/```. It looks to be also used for storing  ```sessions[sessionId] = matches[1]; // store notebook path for later use``` The plan is to go with redirecting for now and open a subsequent issue where some refactoring would have to be done. @parente